packages/components/input-number/src/input-number.vue

```diff
@@ -60,7 +60,14 @@
       @focus="handleFocus"
       @input="handleInput"
       @change="handleInputChange"
-    />
+    >
+      <template #prefix>
+        <slot name="prefix" />
+      </template>
+      <template #suffix>
```

Comment:
Perhaps a judgment should be added here to prevent unnecessary label content from appearing when no slot is added.